### PR TITLE
Add Brazilian Flag to Translations

### PR DESCRIPTION
**Pull request by marcelosdm**
This PR aims to add the Brazilian flag (in a flat design, like the other flags), to translations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 <br/>
 
-Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md) [(![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR**, ![RU](/assets/flags/RU.png)**RU** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)
+Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md) [(![BR](/assets/flags/BR.png)**PT-BR**, ![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR**, ![RU](/assets/flags/RU.png)**RU** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)
```
Comment (BrunoScheufler) [inline README.md]:
```suggestion
Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chinese.md) [(![BR](/assets/flags/BR.png)**BR**, ![ES](/assets/flags/ES.png)**ES**, ![FR](/assets/flags/FR.png)**FR**, ![HE](/assets/flags/HE.png)**HE**, ![KR](/assets/flags/KR.png)**KR**, ![RU](/assets/flags/RU.png)**RU** and ![TR](/assets/flags/TR.png)**TR** in progress!)](#translations)
```
I think we can simplify this to just `BR` 😄

**Comment by i0natan** [comment]
@sagirk Can this get merged?

I noted that @BrunoScheufler is doing some other improvments with @marcelosdm in other threads, not sure if this is related

**Comment by BrunoScheufler** [comment]
@i0natan this one could still have the `PT-BR` part changed to `BR`, but aside from that it's ready to be merged

**Comment by sagirk** [comment]
@marcelosdm Can you please take care of @BrunoScheufler's suggestion above? Just waiting for this one tiny change to land the PR.

**Comment by marcelosdm** [comment]
> @marcelosdm Can you please take care of @BrunoScheufler's suggestion above? Just waiting for this one tiny change to land the PR.

Sorry!! My bad! hahaha

Just commited that change!

Thank you guys!

**Comment by BrunoScheufler** [comment]
@marcelosdm thanks so much 😄 🎉

**Pull request merged**